core/shared/src/main/scala/zio/AbstractZIOPackage.scala

```diff
@@ -1,3 +1,5 @@
+package zio
```

Comment:
Can we keep package object zio with all the stuff where people usually look for it?
It could be done`package object zio extends EitherCompat with ZEnvDefinition` where `ZEnvDefinition` is a trait containing `type ZEnv = ...`.

Reply:
If we go this route we might also just alias ZEnv as
```scala
type ZEnv         = DefaultRuntime#Environment
```
. I think this might be the nicest way to do this

Reply:
I think I like @LGLO's suggestion, since it makes it easier to find stuff for new contributors.

We could definitely do `type ZEnv = ...` (as per [here](https://dotty.epfl.ch/docs/reference/dropped-features/type-projection.html) it looks still valid in Dotty), although it will lead to a circular reference in `DefaultRuntime` so you'd have to revert that cleanup.

Reply:
True. All in all I think @LGLO's suggestion ends up being the cleanest